lib/req-res/status.js

```diff
@@ -11,7 +17,12 @@ module.exports = function ( respond ) {
 		},
 
 		'error' : function ( data, errorData, code ) {
-			console.log( 'Error:', data, errorData, code );
+
+			log.error( data, {
```

Comment:
@openam we can. I added message in our error log but I change the attribute to use `msg` instead of `message`. Using `message` doesn't work maybe conflict with winston